west.yml

```
@@ -140,6 +142,12 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    - name: sdk-homekit
```

```suggestion
    - name: homekit
      repo-path: sdk-homekit    
```

That way, users will be able to `west update homekit`, or `west list homekit`. As is, they have to `west list sdk-homekit`. The name should be a logical name, not reflecting its repository URL like this, in my opinion.

---

Done